elytron/pom.xml

```diff
@@ -89,6 +94,12 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.logmanager</groupId>
+            <artifactId>jboss-logmanager</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Is any part, exception the enum, of the log manager used here?

Reply:
Only the SyslogType enum is used, should I just copy the enum over and remove the dependency?

Reply:
I'll say we should consider it. However since this doesn't rely on the logmanager being the actual log manager we're probably okay.